Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -54,6 +53,8 @@ class Navigator {
         return RoadObjectsStore(navigator.roadObjectStore())
     }()
     
+    var tileStore: TileStore!
```

Comment:
I don't really like using implicitly unwrapped optionals, I think we already had a few crashes because of them. Is there an option not to use them?

Reply:
actually yes. and also it possible to remove other optionals above as all of them are resolved in `init`